modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/test/js/designer/definition-builder/diagram-builder/components/sidebar/Sidebar.js

```diff
@@ -82,4 +95,27 @@ describe('The Sidebar component should', () => {
 		expect(inputName).toHaveValue('node_0');
 		expect(textareaDescription).toHaveValue('');
 	});
+
+	it('Have the correct sections according to each node type', () => {
+		nodeTypes.forEach((type) => {
+			if (type === 'task') {
+				expect(contents[type].sections[0]).toContain('nodeInformation');
+				expect(contents[type].sections[1]).toContain(
+					'assignmentsSummary'
+				);
+				expect(contents[type].sections[2]).toContain(
+					'notificationsSummary'
+				);
+				expect(contents[type].sections[3]).toContain('actionsSummary');
+				expect(contents[type].sections[4]).toContain('timersSummary');
+			}
+			else {
+				expect(contents[type].sections[0]).toContain('nodeInformation');
+				expect(contents[type].sections[1]).toContain(
+					'notificationsSummary'
+				);
+				expect(contents[type].sections[2]).toContain('actionsSummary');
+			}
```

Comment:
Hey @rebsilva, good job! just two things: to avoid code repetition, we can extract the contents[types] sections using destructuring. Also, you can use early return instead of using else in this case, it will make the code cleaner.

```ts
	const {sections} = contents[type];

	if (type === 'task') {
		expect(sections[0]).toContain('nodeInformation');
		expect(sections[1]).toContain('assignmentsSummary');
		expect(sections[2]).toContain('notificationsSummary');
		expect(sections[3]).toContain('actionsSummary');
		expect(sections[4]).toContain('timersSummary');

		return;
	}

	expect(sections[0]).toContain('nodeInformation');
	expect(sections[1]).toContain('notificationsSummary');
	expect(sections[2]).toContain('actionsSummary');
```